python/lance/tests/test_pytorch.py

```diff
@@ -47,8 +47,7 @@ def test_data_loader(tmp_path: Path):
     assert torch.equal(value_batch, torch.tensor([10, 11, 12, 13]))
 
 
-@pytest.mark.skipif(platform.system() != "Linux",
-                    reason="there's a bug on mac")
+@pytest.mark.skipif(platform.system() != "Linux", reason="there's a bug on mac")
```

Comment:
This has already been fixed on mac?

Reply:
Weird, I did not add that in. Not sure where it came from ...

Reply:
Oh nvm it's just formatting. So it just needs to be removed